include/nuttx/devicetree/fdt.h

```diff
@@ -0,0 +1,88 @@
+/****************************************************************************
+ * include/nuttx/devicetree/fdt.h
```

Comment:
should we move to include/nuttx/fdt.h

Reply:
Fdt is a subset of devicetree which is why I structured it this way.  I don't have plans for device tree outside of fdt so if you want I can make it the top level. Thoughts?